resources/british_spellings.json

```diff
@@ -1579,8 +1579,6 @@
     "theorised": "theorized",
     "theorises": "theorizes",
     "theorising": "theorizing",
-    "tonne": "ton",
-    "tonnes": "tons",
```

Comment:
These are still correct if perhaps less widely-used, what's the reason for removing them?

Reply:
Ah, no, I understand - you don't want "ton" to ALWAYS be corrected to "tonne", they're all potentially valid words.

Leaving this here because I was confused by your commit message 😛

Reply:
Yes, that is what I meant 👍